package.json

```diff
@@ -1,78 +1,76 @@
 {
-  "name": "babel.github.io",
-  "private": true,
-  "version": "1.0.0",
-  "description": "",
-  "main": "",
-  "scripts": {
-    "start": "make serve",
-    "build": "make build",
-    "lint": "npm run lint:docs && npm run lint:repl",
-    "lint:docs": "eslint --ext md --rule 'prettier/prettier: 0' docs",
-    "lint:repl": "eslint js/repl/*.js",
-    "test": "remark _posts docs _includes README.md --quiet",
-    "flow": "flow",
-    "precommit": "flow && lint-staged"
-  },
-  "remarkConfig": {
-    "plugins": {
-      "preset-lint-recommended": true,
-      "lint-list-item-indent": false,
-      "lint-no-blockquote-without-caret": false,
-      "lint-no-literal-urls": false,
-      "lint-no-duplicate-headings-in-section": true,
-      "lint-no-empty-url": true
+    "name": "babel.github.io",
+    "private": true,
+    "version": "1.0.0",
+    "description": "",
+    "main": "",
+    "scripts": {
+        "start": "make serve",
+        "build": "make build",
+        "lint": "npm run lint:docs && npm run lint:repl",
+        "lint:docs": "eslint --ext md --rule 'prettier/prettier: 0' docs",
+        "lint:repl": "eslint js/repl/*.js",
+        "test": "remark _posts docs _includes README.md --quiet",
+        "flow": "flow",
+        "precommit": "flow && lint-staged"
+    },
+    "remarkConfig": {
+        "plugins": {
+            "preset-lint-recommended": true,
+            "lint-list-item-indent": false,
+            "lint-no-blockquote-without-caret": false,
+            "lint-no-literal-urls": false,
+            "lint-no-duplicate-headings-in-section": true,
+            "lint-no-empty-url": true
+        }
+    },
+    "author": "",
+    "license": "ISC",
+    "devDependencies": {
```

Comment:
Please revert all of the white-space only changes to this file if possible. 😄 

So far as I can see, the only 2 _actual_ changes to this file are:
* The addition of the `prettysize` module
* An (unnecessary?) formatting change to the `lint-staged` block

Reply:
This still needs to be reverted.